docs/Reference.md

```diff
@@ -265,6 +265,8 @@ Example:
 />
 ```
 
+The `syntheticEvent` can be stopped doing its default action by calling `syntheticEvent.preventDefault()`.
```

Comment:
I'd probably put this at the end of this section (just before the ### `onLoad` line).

Reply:
Updated it according to your suggestion.